src/implementations/vanilla/components/gallery/gallery.js

```diff
@@ -407,29 +423,39 @@ export class Gallery {
         this.overlayMedia.innerHTML = '';
         this.overlayMedia.appendChild(mediaElement);
       }
-    }
 
-    // Limit image height (fix for FF and IE)
-    const maxHeight =
-      this.overlay.clientHeight -
-      this.overlayHeader.clientHeight -
-      this.overlayFooter.clientHeight;
-    if (this.overlayMedia) {
-      Object.assign(mediaElement.style, {
-        maxHeight: `${maxHeight}px`,
-      });
+      // DEPRECATED
```

Comment:
Is the deprecated part of the code carried for a specific reason? I had the impression we use v3 for removing deprecated code.

Reply:
Thank you I didn't pay attention to this deprecated code. Tested and removed.